parser/parser_test.go

```diff
@@ -365,6 +365,62 @@ world`},
 			&UnaryNode{Operator: "not",
 				Node: &IdentifierNode{Value: "in_var"}},
 		},
+		{
+			"-1 not in [1, 2, 3, 4]",
+			&UnaryNode{Operator: "not",
+				Node: &BinaryNode{Operator: "in",
+					Left: &UnaryNode{Operator: "-", Node: &IntegerNode{Value: 1}},
+					Right: &ArrayNode{Nodes: []Node{
+						&IntegerNode{Value: 1},
+						&IntegerNode{Value: 2},
+						&IntegerNode{Value: 3},
+						&IntegerNode{Value: 4},
+					}}}},
+		},
+		{
+			"1*8 not in [1, 2, 3, 4]",
+			&UnaryNode{Operator: "not",
+				Node: &BinaryNode{Operator: "in",
+					Left: &BinaryNode{Operator: "*",
+						Left:  &IntegerNode{Value: 1},
+						Right: &IntegerNode{Value: 8},
+					},
+					Right: &ArrayNode{Nodes: []Node{
+						&IntegerNode{Value: 1},
+						&IntegerNode{Value: 2},
+						&IntegerNode{Value: 3},
+						&IntegerNode{Value: 4},
+					}}}},
+		},
+		{
+			"2==2 ? false : 3 not in [1, 2, 5]",
```

Comment:
Will be cool to require a brackets `(` `)` in this case too. As even for me, it's unclear what operator will be applied first.

Reply:
```
2 == 2 ? false : (3 not in [1, 2, 5])
```

But we can implement it in a new PR.

Reply:
This behavior aligns with expectations since the ':' operator is not considered a comparison operator.